examples/llava/clip.cpp

```diff
@@ -2990,6 +2983,31 @@ bool clip_is_qwen2vl(const struct clip_ctx * ctx) {
     return ctx->has_qwen2vl_merger;
 }
 
+size_t get_max_image_grid_pinpoints() {
+    return MAX_IMAGE_GRID_PINPOINTS;
+}
+
+// Determine the number of encoder layers to iterate over
+CLIP_API int get_deepest_feature_layer(const struct clip_ctx * ctx) {
```

Comment:
I think you want to remove the `CLIP_API` here

Reply:
Yup, nice catch!